### PR TITLE
Fix test task: handle test tags and modules to skip correctly

### PR DESCRIPTION
**Pull request by joao-p-marques**

### CODE REVIEW

File: tasks_downstream.py [hunk 1]
```diff
@@ -651,6 +651,7 @@ def test(
         modules = cur_module
     else:
         modules = _get_module_list(c, modules, core, extra, private)
+    skip = skip or ""
```
Comment (Yajo) [inline tasks_downstream.py:654]: Just set `""` as `skip`'s default value instead of `None`

File: tasks_downstream.py [hunk 2]
```diff
@@ -660,16 +661,18 @@ def test(
         raise exceptions.ParseError(
             msg="Available modes are 'init' or 'update'. See --help for details."
         )
-    odoo_command.append(modules)
     # Skip test in some modules
     modules_list = modules.split(",")
-    for m_to_skip in skip or []:
+    for m_to_skip in skip.split(","):
+        if not m_to_skip:
+            continue
```
Comment (Yajo) [inline tasks_downstream.py:668]: Passing in `--skip ,,,` should be supported? 🤔

Comment (joao-p-marques) [reply tasks_downstream.py:668]: Well, this was covering the case where `skip` is `""` (default) -> the list will be `[""]` -> the first `m_to_skip` will be `""`, which shouldn't be parsed. Is it worth preventing that specific situation and throw an error? `--skip ,,,` won't cause damage, just won't be parsed...

**Comment by joao-p-marques** [comment]
@Yajo just added a new test to assert that (the skip part was not being tested)

**Review by Yajo — APPROVED** [review]

**Pull request merged**